src/Discord.Net.Commands/Results/RuntimeResult.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace Discord.Commands
+{
+    [DebuggerDisplay(@"{DebuggerDisplay,nq}")]
+    public class RuntimeResult : IResult
+    {
+        private RuntimeResult(CommandError? error, string reason)
```

Comment:
:upside_down_face: I'm sure I managed to subclass this when I tested it originally, good catch though.